backend/app/controllers/api/v1/investors_controller.rb

```diff
@@ -32,6 +33,12 @@ def fetch_investor
         account = Account.friendly.find(params[:id])
         Investor.find_by!(account_id: account.id)
       end
+
+      def filter_params
+        params.fetch(:filter, {})
+          .permit :category, :impact, :sdg, :instrument_type, :ticket_size, :only_verified,
+            category: [], impact: [], sdg: [], instrument_type: [], ticket_size: []
```

Comment:
So to filter by multiple request like this is needed `http://localhost:4000/backend/api/v1/investors?filter[category][]=tourism-and-recreation&filter[category][]=forestry-and-agroforestry`, I would probably just try to do something like `filter[category]=cat1,cat2,cat3` but maybe that is ok, what do you think @agnessa ?

Reply:
Maybe that is just a detail, and the current implementation at least does not have to do anything special with params and just feeding the Filterer service.

Reply:
Actually, that is good point. I normally prefer array option, even though url looks more ugly, but we already have some params which use `,` to split multiple options and it will solve rswag doc issue, so why not ;)